### PR TITLE
added ability to upgrade cards in parallel

### PR DESCRIPTION
**Pull request by jokaorgua**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -2,21 +2,16 @@
...
 
+import asyncio
 import datetime
-import requests
 import json
-import time
 import logging
-import asyncio
 import random
-import base64
```
Comment (masterking32) [inline main.py:12]: Why did you remove this line?

File: main.py [hunk 2]
```diff
@@ -2,21 +2,16 @@
 # Date: 2024
 # Github: https://github.com/masterking32
 
+import asyncio
 import datetime
-import requests
```
Comment (masterking32) [inline main.py:6]: Why did you remove this line?

File: main.py [hunk 3]
```diff
@@ -56,6 +51,7 @@
             # This feature will stop buying upgrades when the balance is less than the price of the best card.
             "wait_for_best_card": False,  # Recommended to keep it True for high level accounts
             "auto_get_task": True,  # Enable auto get (Youtube/Twitter and ...) task to True, or set it to False to disable
+            "parallel_upgrades_max_price_per_hour": 6_000_000,  # Enable parallel card buy if 1k of profit costs less than value
```
Comment (masterking32) [inline main.py:54]: Need to add a better description with more detail and also support 0 for disabling the feature.
Also will be good, if you allow them to limit loops, like 2-3 or 5 or etc ...

**Review by masterking32 — CHANGES_REQUESTED** [review]
Hello, Thank you, Can you check these changes?

File: main.py (hunk 1 above)
Comment (jokaorgua) [reply main.py:12]: it is imported in utilities

File: main.py (hunk 2 above)
Comment (jokaorgua) [reply main.py:6]: it was imported in utilities but not used. moved to main

File: main.py (hunk 3 above)
Comment (jokaorgua) [reply main.py]: added a flag to disable-enable feature.

I don't think it is a good idea to limit loops. I do not see benefits in such a behavior

**Review by masterking32 — APPROVED** [review]
Cool thanks.
